client.py

```diff
@@ -58,6 +60,8 @@ def __init__(self, query_text):
             "original_url",
         ]
         self._expanded_source = self._source + ["text_content", "text_extraction"]
+        if client_config.debug:
+            logger.info(f"Building es query for {self.query_text}")
```

Comment:
If we're checking `debug` flag, should we also use `logger.debug`?

Reply:
yes, fair!